platform/internal/SysTimer.cpp

```diff
@@ -0,0 +1,318 @@
+/*
```

Comment:
Okay, I can adjust patchset so it moves into platform first, so you'll see a clearer diff on single commit view.

You want it to just go into `platform/SysTimer.[cpp|h]` then? As we don't have separate include directories yet, it seemed wise to otherwise keep it apart from the API headers. (It currently is a bit tucked away in `rtos/TARGET_CORTEX`, so doesn't just visibly sit alongside all the other `rtos/Mutex.h`)

Reply:
Oh, and note that `mbed_atomic_impl.[c|h]` are already in `platform/internal` from #10147, so that split has already happened on master.

Reply:
> I don't see any problem with having them split like this. I don't see the logic in keeping private stuff with public now just because there's a reorganisation planned. Is it because you're planning /platform/SysTimer.h to be the final private position post-reorg?

I would leave it as well, but good note about `include` directory. We will move internal one accordingly to the new set of rules.